programs/ziti-edge-tunnel/instance.c

```diff
@@ -354,7 +382,9 @@ tunnel_identity_array get_tunnel_identities() {
 
     int idx = 0;
     MODEL_MAP_FOREACH(id, tnl_id, &tnl_identity_map) {
-        tnl_id_arr[idx++] = tnl_id;
+        if (tnl_id->Status != NULL && strcmp(tnl_id->Status, instance_status_name(instance_status_ok)) == 0) {
```

Comment:
change tnl_id.Status to instance_status enum, then you don't need to use strcmp()

Reply:
changed the status field